### PR TITLE
Added mango phytochemicals to the Node

### PR DESCRIPTION
**Pull request by Nickspizza001**

### CODE REVIEW

File: global_chem/global_chem/global_chem.py [hunk 1]
```diff
@@ -472,6 +486,12 @@ class GlobalChem(object):
         'lanthipeptides': LanthiPeptides,                                                # Prabin Baral & Suliman Sharif
         'alternative_jet_fuels': AlternativeJetFuels,                                    # Suliman Sharif
         'common_regex_patterns': CommonRegexPatterns,                                    # Chris Burke & Suliman Sharif
+        'mango_phytocompounds': Mango,                                                   # Damilola Bodun & Sevien Schulhofff
+        'mango_amino_acids': MangoAminoAcids,                                            # Damilola Bodun & Sevien Schulhofff
+        'mango_phenolic_acids': MangoPhenolicAcids,                                      # Damilola Bodun & Sevien Schulhofff
+        'mango_fatty_acids': MangoFattyAcids,                                            # Damilola Bodun & Sevien Schulhofff
+        'mango_vitamins': MangoVitamins,                                                 # Damilola Bodun & Sevien Schulhofff
+        'mango_flavonoids': MangoFlavonoids                                              # Damilola Bodun & Sevien Schulhofff
```
Comment (Sulstice) [inline global_chem/global_chem/global_chem.py:494]: add my name too :)

File: global_chem/global_chem/global_chem.py [hunk 2]
```diff
@@ -111,6 +111,20 @@
 from global_chem.food.color_additives.fda_list_six import FDAListSix
 from global_chem.food.color_additives.fda_list_seven import FDAListSeven
 
+
+
+# Food/Fruits/Mango
+
+
+from global_chem.food.fruits.mango.constituents_of_mango import Mango
```
Comment (Sulstice) [inline global_chem/global_chem/global_chem.py:119]: Can you add these imports to the __init__ as well this way the GlobalChem module can see the rest of the modules:

https://github.com/Global-Chem/global-chem/blob/development/global_chem/global_chem/__init__.py

**Review by Sulstice — APPROVED** [review]

File: global_chem/global_chem/global_chem.py (hunk 2 above)
Comment (Nickspizza001) [reply global_chem/global_chem/global_chem.py:119]: Fixed Alreay, You can please check through

File: global_chem/global_chem/global_chem.py (hunk 1 above)
Comment (Nickspizza001) [reply global_chem/global_chem/global_chem.py:494]: Yeahhh, For sure. Did that!

**Pull request merged**